### PR TITLE
Push one podcast to another

### LINKED ISSUE #140: refactor longer list to be a push from the long list, copying to the inbox of the selected podcast(s)
the way i'd envisaged this to work:

the 'longerlist' implementation is wrong, so can be trashed
instead:
you should be able to **push** from one podcast to another
so a new collection on each podcast there should be an `inbox` collection, which should only require write access
therefore you should be able to push from the `dev` podcast to the `dev2` podcast's inbox
so `dev2.inbox[hash_of_text].value`
`hash_of_text` is a hash of the text, used to avoid duplication.
this should be written as a document so that you should not depend on `read` access, it should be able to operate like a blind dropbox, you throw it over the wall.

this should be a copy, theres no relationship maintained between the original item and the copy, it is likely that the user will not have `read` access to the destination podcast.

note I've currently configured the firebase rules to look like:
```
rules_version = '2';
service cloud.firestore {
  match /databases/{database}/documents {
    match /{document=**} {
      allow read, write: if
      request.auth.token.email_verified &&
      request.auth.token.email in get(/databases/$(database)/documents/smartseven/members).data.email;
    }
  }
  match /databases/{database}/documents/dev {
    match /{document=**} {
      allow read, write: if request.auth != null;
    }
  }
  match /databases/{database}/documents/dev2 {
    match /{document=**} {
      allow read, write: if request.auth != null;
    }
  }
  match /databases/{database}/documents/dev3/inbox {
    match /{document=**} {
      allow write: if request.auth != null;
    }
  }
}
```

so you should be able to write to `dev3/inbox/*` but not be able to read from it, but you can read and write to `dev1` and `dev2`

this is a fair chunk of work, so I'd encourage sketching it out from a high level before you do anything else and schedule time with me to discuss if you want before investing tonnes of time in writing code.

## note:
the scope of this work is **not to then present this back to the user**, it is simply to get the stuff written to firebase correctly, further issues describe the usability of the data once it is in the inbox.

I did start work on this:
https://github.com/DaftDoris/newsdesk/tree/inbox
but I've not had time to progress it, feel free to look at that if its useful, or dismiss it, I can't even really remember how far I got, it's probably not useful, but its there if you need it

### PR DESCRIPTION
**Pull request by bhavinsen**
Fixed #140

### CODE REVIEW

**Comment by mrjamieeast** [comment]
This works as expected - Chris if you're happy = i am.

File: src/pages/Home.vue [hunk 1]
```diff
@@ -203,8 +209,22 @@ const events = {
...
-    item.shared = !item.shared
+  async onClickShare(item: Item, podcastNameToShare: any, deletePodcastId = "", AddPodcastId = "") {
+    if (deletePodcastId) {
+      await shareStore.deteleSendItem(item, docname, deletePodcastId)
+    } else {
+      await shareStore.sendItem(item, docname, AddPodcastId)
+      //  if(getErrorStatus.value){
+      //   const index = podcastNameToShare.findIndex((x: string) => x === AddPodcastId)
+      //   podcastNameToShare.splice(index, 1)
+      // }
```
[the remark this answers is not in the document]
Comment (bhavinsen) [reply src/pages/Home.vue]: I have removed commented code

**Comment by chrisns** [comment]
as per notes in #140
- [ ] shared items should be their own document
- [ ] unsharing consequently isn't possible
- [ ] "longerlist" is no longer a thing
- [ ] sharing should never mention
- [ ] you shouldn't rely on being able to read the contents of a shared item, its a 'dead drop' type thing
- [ ] it shouldn't make any reference to "todaysdate" there is a later piece of work to have multiple dates, which is why inbox/shared items shouldn't have any connection to the docname of the podcast where the item originated

please re-read #140 perhaps it'd be helpful for you to playback the requirements as you see them before you commence work on it

**Comment by chrisns** [comment]
closed in favour of #204

**Pull request closed without merging**